File: ARRAY cpp/bubble_sort.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;
    vector<int> array(n);

    for (int i = 0; i < n; i++) {
        cin >> array[i];
    }

    // Bubble sort
    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - 1 - i; j++) {
            if (array[j] > array[j + 1]) {
                swap(array[j], array[j + 1]);
            }
        }
    }

    for (int i = 0; i < n; i++) {
        cout << array[i] << endl;
    }
}
///https://www.w3schools.com/dsa/dsa_algo_bubblesort.php

File: ARRAY py/bubble_sort.py
list =[]
n=int(input("enter number of input: "))

for _ in range(n):
    x=int(input("_>"))
    list.append(x)

for i in range(n-1):
    for j in range(i+1,n):
        if list[i]>list[j]:
            list[i],list[j]=list[j],list[i]

print(*list)

##https://www.w3schools.com/dsa/dsa_algo_bubblesort.php